packages/grafana-data/src/dataframe/MutableDataFrame.ts

```diff
@@ -115,6 +115,8 @@ export class MutableDataFrame<T = any> extends FunctionalVector<T> implements Da
       config: f.config || {},
       values: this.creator(buffer),
       labels: f.labels,
+      display: f.display,
+      getLinks: f.getLinks,
```

Comment:
This is very ad-hoc, to be changed. After the transform when aligning data frames the display processor and get links need to be attached back to the fields